assets/new-user-experience/onboarding/data/reducers.js

```diff
@@ -0,0 +1,201 @@
+export const CREDIT_CARD_DEFAULT_STATE = {
+	enabled: 'yes',
+	title: 'Credit Card',
```

Comment:
There's a decent amount of text in this file that isn't being translated. I've not parsed everything yet so maybe that doesn't matter but flagging that as something to consider if needed.